src/protocol/amcp/AMCPCommandsImpl.cpp

```diff
@@ -1342,7 +1343,10 @@ std::wstring info_channel_command(command_context& ctx)
 
     auto state = ctx.channel.channel->state();
     for (const auto& p : state) {
-        const auto    path = boost::algorithm::replace_all_copy(p.first, "/", ".");
+        const auto replaced = boost::algorithm::replace_all_copy(p.first, "/", ".");
+        // avoid digit-only nodes in XML
+        const auto path  = boost::algorithm::replace_all_regex_copy(
+            replaced, boost::regex("\\.(.*?)\\.([0-9]*?)\\."), boost::lexical_cast<std::string>(".$1.$1_$2."));
```

Comment:
Not sure what this does. Example?

Reply:
I gave the example in the PR? 
>Number-only nodes are not allowed in the XML spec so the INFO-command is currently breaking some XML-parsers. I've changed this so that the preceding name will be appended to the number. For example <layer><0></0></layer> will now be <layer><layer_0></layer_0></layer>